README.md

```diff
@@ -16,82 +20,811 @@ yarn add gotrue-js
+    "token_type": "bearer",
+    "expires_in": 3600,
+    "refresh_token": "example-refresh_token",
+    "expires_at": 1525214326000
+  },
+  "id": "example-id",
+  "aud": "",
+  "role": "",
+  "email": "example@example.com",
+  "confirmed_at": "2018-05-01T19:21:00Z",
+  "app_metadata": {
+    "provider": "email"
+  },
+  "user_metadata": {},
+  "created_at": "2018-05-01T19:21:00Z",
+  "updated_at": "2018-05-01T19:21:00Z"
+}
+```
+
+### Update a user
+
+This function updates a user object with specified attributes
+
+`user.update( attributes )`
+
+Example usage:
+
+```js
+const user = auth.currentUser();
+
+user
+  .update({ email: "example@example.com", password: "password" })
+  .then(user => console.log("Updated user %s", user))
+  .catch(error => {
+    console.log("Failed to update user: %o", error);
+    throw error;
+  });
+```
+
+Example response object:
+
+```js
+{
+  "api": {
+    "apiURL": "https://example.netlify.com/.netlify/identity",
+    "_sameOrigin": true,
+    "defaultHeaders": {}
+  },
+  "url": "https://example.netlify.com/.netlify/identity",
+  "token": {
+    "access_token": "example-jwt-token",
+    "token_type": "bearer",
+    "expires_in": 3600,
+    "refresh_token": "example-refresh_token",
+    "expires_at": 1525215471000
+  },
+  "id": "example-id",
+  "aud": "",
+  "role": "",
+  "email": "example@example.com",
+  "confirmed_at": "2018-05-01T19:21:00Z",
+  "app_metadata": {
+    "provider": "email"
+  },
+  "user_metadata": {},
+  "created_at": "2018-05-01T19:21:00Z",
+  "updated_at": "2018-05-01T22:04:07.923944421Z",
+  "new_email": "new-example@example.com",
+  "email_change_sent_at": "2018-05-01T22:04:07.49197052Z"
+}
+```
+
+### Get a JWT token
+
+This function retrieves a JWT token from a currently logged in user
+
+`user.jwt()`
+
+Example usage:
+
+```js
+const jwt = user.jwt();
+jwt
+  .then(response => console.log("This is a JWT token", response))
+  .catch(error => {
+    console.log("Error fetching JWT token", error);
+    throw error;
+  });
+```
+
+Example response object:
+
+```
+eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1MjUyMTk4MTYsInN1YiI6ImE5NG.98YDkB6B9JbBlDlqqef2nme2tkAnsi30QVys9aevdCw debugger eval code:1:43
+```
+
+### Logout a user
+
+This function removes the current session of the user and log out the user
+
+`user.logout()`
+
+Example usage:
+
+```js
+const user = auth.currentUser();
+user
+  .logout()
+  .then(response => console.log("User logged out");)
+  .catch(error => {
+    console.log("Failed to logout user: %o", error);
+    throw error;
+  });
+```
+
+## Admin methods
+
+The following admin methods are currently not available to be used directly. You can access `context.clientContext.identity` and get a short lived admin token through a Lambda function and achieve the same goals, e.g., update user role, create or delete user etc. See [Identity and Functions](https://www.netlify.com/docs/functions/#identity-and-functions) for more info.
+
```

Comment:
Very awkward.  We have lib that half works in the browser and half works in a node/lambda context.  We should probably break this up into two separate modules, or make the lib not assume local storage and work in a lambda context.

Reply:
That was the most confusing part to me :p